core/primitives/src/offchain.rs

```diff
@@ -204,22 +204,22 @@ impl Timestamp {
 	}
 
 	/// Increase the timestamp by given `Duration`.
-	pub fn add(&self, duration: Duration) -> Timestamp {
+	pub fn plus(self, duration: Duration) -> Timestamp {
```

Comment:
I prefer before

Reply:
Ok! I'll just silence this, and the other warnings as well (for `sub` and `mul`).